auth0/v3/authentication/revoke_token.py

```diff
@@ -0,0 +1,36 @@
+from .base import AuthenticationBase
+
+
+class RevokeToken(AuthenticationBase):
+
+    """Revoke Refresh Token endpoint
+
+    Args:
+        domain (str): Your auth0 domain (e.g: username.auth0.com)
+    """
+
+    def revoke_refresh_token(self, client_id, token, client_secret=None):
+        """Revokes a Refresh Token if it has been compromised
+           
+           Each revocation request invalidates not only the specific token, but all other tokens 
+           based on the same authorization grant. This means that all Refresh Tokens that have 
+           been issued for the same user, application, and audience will be revoked.
+
+           Args:
+                client_id (str): your application's client Id
+
+                token (str): the Refresh Token you want to revoke
+
+                client_secret (str, optional): Your application's Client Secret. 
+                        Required for confidential applications.
```

Comment:
This is confusing ... what is a "confidential application?" This doesn't tell me much about when I should be using this.